### ISSUE TITLE
macOS Catalina support?

### ISSUE THREAD

**Issue by gandalfsaxe** [danielecook/alfred-image-utilities#2]
Any chance this could be made to work out-of-the-box on macOS Catalina?

**Comment by danielecook** [comment]
I'll have to take a look - I wasn't aware it didn't work in Catalina. Is there a certain error you are seeing?

**Comment by gandalfsaxe** [comment]
First I get this:
![image](https://user-images.githubusercontent.com/12127864/76084319-ea1e6f00-5faf-11ea-8b8a-518a212b6925.png)

Clicking the link goes to this (Fixum): https://www.alfredapp.com/help/troubleshooting/incompatible-python-workflow-library/

After running Fixum to fix Alfred Image Utilities, I now see the workflow:

![image](https://user-images.githubusercontent.com/12127864/76084379-0de1b500-5fb0-11ea-95df-411b2d648145.png)

Finally, running e.g. a scale of 50% on an image, I finally get this:
![image](https://user-images.githubusercontent.com/12127864/76084420-25b93900-5fb0-11ea-9842-4d2038367734.png)

So in summary:
1. There is a minor problem with python libaries not being compatible, and while this should be fixed, it can be fixed with Fixum
2. `_imaging.so` is not trusted. Perhaps there is a way to rebundle it differently, perhaps not as a .so file? Otherwise I think the only way is to sign it as an apple developer, but that costs money.

**Comment by danielecook** [comment]
@gandalfsaxe I'm on OS X Catalina and just tried the workflow out and it worked fine.... 

Is there a way to get around this in the user settings?

One think you may also try is to just clone the repo from github to your workflow folder maybe?

**Comment by gandalfsaxe** [comment]
I just tried on another Mac machine with macOS Catalina (10.15.3), same problem. I tried right clicking on `_imaging.so` and click "open", but it didn't make a difference.

I tried cloning/download the contents of the repo directly to workflows folder instead of running the `Alfred.Image.Utilities.alfredworkflow` file. The workflow correctly showed up in Alfred preferences, but didn't do anything when I an operation on a photo. No error, but no new photo either.

**Comment by danielecook** [comment]
Maybe if we remove the .so file python will rebuild it...

**Comment by gandalfsaxe** [comment]
Unfortunately not:

![image](https://user-images.githubusercontent.com/12127864/80283679-d246aa00-8719-11ea-8ccd-2069fa36ba84.jpeg)


```
[17:25:22.422] ERROR: Alfred Image Utilities[Run Script] Traceback (most recent call last):
  File "convert.py", line 3, in <module>
    from PIL import Image
  File "/Users/gandalf/Documents/-sync/Alfred/Alfred.alfredpreferences/workflows/user.workflow.B86CC126-66D2-4927-AD95-1141DE127FC4/PIL/Image.py", line 67, in <module>
    from PIL import _imaging as core
ImportError: cannot import name _imaging
```

**Comment by richardsison** [comment]
Cool… not just me. Any update on this? 🙏

**Comment by zeshanshani** [comment]
Any update on this yet? I used Fixum workflow to fix the pyton support for catalina. But now it just generates a blank image.
